githooks/install.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
```

Comment:
Does every developer need to run this script? If yes please add instructions to the developer guide.